docs/installation_configuration_guide/DeploymentManagerConfiguration.md

```diff
@@ -169,45 +188,74 @@ modelConfig {
+Kubernetes label.
+In a standard scenario, its value is taken from Nussknacker's pod `app.kubernetes.io/instance` label which, when
+installed
 using helm should be set to [helm release name](https://helm.sh/docs/chart_best_practices/labels/#standard-labels).
 
 It can be used to identify scenario deployments and its resources bound to a specific Nussknacker helm release.
 
-&nbsp;
 ### Configuring runtime logging
-With `logbackConfigPath` you can provide path to your own logback config file, which will be used by runtime containers. This configuration is optional, if skipped default logging configuration will be used. 
-Please mind, that apart whether you will provide your own logging configuration or use default, you can still modify it in runtime (for each scenario deployment separately*) as described [here](../operations_guide/Lite#logging-level)
 
-*By default, every scenario runtime has its own separate configMap with logback configuration. By setting `commonConfigMapForLogback` you can enforce usage of single configMap (with such name as configured) with logback.xml for all your runtime containers. Take into account, that DeploymentManager relinquishes control over lifecycle of this ConfigMap (with one exception - it will create it, if not exist).
+With `logbackConfigPath` you can provide path to your own logback config file, which will be used by runtime containers.
+This configuration is optional, if skipped default logging configuration will be used.
+Please mind, that apart whether you will provide your own logging configuration or use default, you can still modify it
+in runtime (for each scenario deployment separately*) as described [here](../operations_guide/Lite#logging-level)
+
+*By default, every scenario runtime has its own separate configMap with logback configuration. By
+setting `commonConfigMapForLogback` you can enforce usage of single configMap (with such name as configured) with
+logback.xml for all your runtime containers. Take into account, that DeploymentManager relinquishes control over
+lifecycle of this ConfigMap (with one exception - it will create it, if not exist).
+
+### Configuring runtime ingress
+
+In Request-Response processing mode additional ingress resource can be created when `enabled` flag is turned on.
+For now only [nginx based](https://docs.nginx.com/nginx-ingress-controller/) K8s ingress controller is supported.
+It can be configured with following options.
+
+| Parameter | Type    | Default value | Description                                                                                                                   |
+|-----------|---------|---------------|-------------------------------------------------------------------------------------------------------------------------------|
+| enabled   | boolean |               | Either streaming or request-response                                                                                          |
```

Comment:
its `ingress.enabled`, `ingress.host` etc. If you do not want to have this prefix in the table, can you somehow add this information in the description?